File: InSeoul/app/src/main/java/com/inseoul/register_review/review_request.java
package com.inseoul.register_review;

import com.android.volley.Response;
import com.android.volley.toolbox.StringRequest;

import java.util.HashMap;
import java.util.Map;

public class review_request extends StringRequest {
    final static private String URL= "http://ksun1234.cafe24.com/ReviewRegister.php";
    private Map<String, String> parameters;

    public review_request(int PLANID, int SCORE, String IMAGENAME, String USERREVIEW, Response.Listener<String> listener){

        super(Method.POST, URL, listener, null);
        parameters=new HashMap<>();
        parameters.put("PLANID",Integer.toString( PLANID));
        parameters.put("SCORE", Integer.toString( SCORE));
        parameters.put("IMGNAME", IMAGENAME);
        parameters.put("REVIEW",USERREVIEW);




    }
    @Override
    public Map<String, String>getParams(){


        return parameters;
    }


}

File: InSeoul/app/src/main/java/com/inseoul/register_review/review_request.kt
package com.inseoul.register_review

import com.android.volley.Request
import com.android.volley.Response
import com.android.volley.toolbox.StringRequest

import java.util.HashMap

class review_request(
    PLANID: Int,
    SCORE: Int,
    IMAGENAME: String,
    USERREVIEW: String,
    listener: Response.Listener<String>
) : StringRequest(Request.Method.POST, URL, listener, null) {
    private val parameters: MutableMap<String, String>

    init {
        parameters = HashMap()
        parameters["PLANID"] = Integer.toString(PLANID)
        parameters["SCORE"] = Integer.toString(SCORE)
        parameters["IMGNAME"] = IMAGENAME
        parameters["REVIEW"] = USERREVIEW


    }

    public override fun getParams(): Map<String, String> {


        return parameters
    }

    companion object {
        private val URL = "http://ksun1234.cafe24.com/ReviewRegister.php"
    }


}